tests/components/risco/test_config_flow.py

```diff
@@ -42,6 +44,19 @@
 }
 
 
+async def _configure_with_exception(hass, flow_id, exception):
```

Comment:
Instead of breaking out a test helper, use pytest parametrization to parametrize the test cases of a single test function.

https://docs.pytest.org/en/stable/how-to/parametrize.html

This makes it easier to read and understand the test, since we keep the test detail logic in one place.

Reply:
Cool, done!